noxfile.py

```diff
@@ -54,6 +54,12 @@ def check_newsfragment(session: nox.Session) -> None:
     session.run("python", "-m", "towncrier.check", "--compare-with", "origin/trunk")
 
 
+@nox.session
+def draft_newsfragment(session: nox.Session) -> None:
```

Comment:
I went with nox, but I don't realy think we need this and check_newsfragment 

but we can keep them for now